x-pack/legacy/plugins/uptime/public/components/monitor_details/ml/ml_flyout.tsx

```diff
@@ -64,11 +65,19 @@ export function MLFlyoutView({ isCreatingJob, onClickCreate, onClose, canCreateM
             <em>{labels.TAKE_SOME_TIME_TEXT}</em>
           </p>
         </EuiText>
-
         <EuiSpacer />
       </EuiFlyoutBody>
       <EuiFlyoutFooter>
         <EuiFlexGroup justifyContent="spaceBetween" alignItems="flexEnd">
+          <EuiFlexItem grow={true}>
```

Comment:
```suggestion
          <EuiFlexItem grow={false}>
```

Reply:
if i set this to false, create new button will take all the space.

Reply:
i need to remove alignItems="flexEnd"